app/views/results/exceptionsResults.scala.html

```diff
@@ -0,0 +1,97 @@
+@*
+* Copyright 2016 LinkedIn Corp.
+*
+* Licensed under the Apache License, Version 2.0 (the "License"); you may not
+* use this file except in compliance with the License. You may obtain a copy of
+* the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+* WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+* License for the specific language governing permissions and limitations under
+* the License.
+*@
+
+@(result: com.linkedin.drelephant.exceptions.HadoopException)
+
+@import com.linkedin.drelephant.exceptions.HadoopException
+
+
+
+@tags.panel() { Exception Results} {
+  <head>
+    <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
```

Comment:
I think we already have these libraries in Dr. Elephant. You need not fetch them from the network.

```Check main.scala.html file for examples.```